### DISCUSSION TITLE
Issue with Field Name Mapping in `nestjs-graphql-tools`

### DISCUSSION THREAD

**Discussion by dagyu** [Adrinalin4ik/Nestjs-Graphql-Tools#36] [Q&A] [answered]
### Introduction
I'm using the `nestjs-graphql-tools` package and have been really impressed with its capabilities. It's been an invaluable tool in my projects. However, I've come across a specific issue regarding field name mapping that I'd like to address.

### Environment
- NestJS Version: [Specify version]
- `nestjs-graphql-tools` Version: 0.9.0
- Database Type: [e.g., PostgreSQL, MySQL]

### Problem
When defining an entity with different field names in the code and the database, the query generated by `nestjs-graphql-tools` incorrectly uses the code field name instead of the database field name.

### Example Entity
```js
@ObjectType()
@Entity()
class Test {
   @Field(() => Date)
   @Column({name: "start_at"})
   startAt: Date;
}
```

### Issue
Applying a filter like `@Filter(() => Test) filter: FilterArgs` produces a query using startAt as the field name, whereas it should use start_at.

### Current workaround

```js
@ObjectType()
@Entity()
class Test {
   @Field(() => Date)
   @FilterField(() => Date, { sqlExp: 'test.start_at' })
   @Column({name: "start_at"})
   startAt: Date;
}
```

This workaround involves hard-coding the entity name, which seems unsafe.

### Question
Is there a better solution to ensure the correct field name (as defined in the database) is used in queries, without resorting to hard-coded values?

### Expected Behavior
The nestjs-graphql-tools should automatically use the database field name (start_at in this case) in the generated queries when a filter is applied, without the need for custom filters or hard-coded values.

**Comment by Adrinalin4ik** [comment, marked as answer]
@dagyu Hello! I see your problem.  At the current state it's the only way how you can do that. If you see any other way, feel free to open PR. 
I would recommend not to use mappings like you do. Instead of specifying @Column({name: "start_at"}), just use start_at instead of startAt.
